dotnet-authserver/src/TeacherIdentity.AuthServer/Services/Redactor.cs

```diff
@@ -55,4 +55,14 @@ public class Redactor
 
         return builder.ToString();
     }
+
+    public string? RedactMobileNumber(string mobileNumber)
+    {
+        if (string.IsNullOrEmpty(mobileNumber))
+        {
+            return string.Empty;
+        }
+
+        return new string('*', mobileNumber.Length - 4) + mobileNumber.Substring(mobileNumber.Length - 4);
```

Comment:
Would be good to clone the implementation we're already using elsewhere: https://github.com/DFE-Digital/dqt-web-portal/blob/3811d7449b34f318d27f9e250f76a9181e890f1d/src/Dqt.Portals/Capgemini.DfE.Evolved.Model/RedactionHelper.cs#L71

Reply:
Would that work for international numbers? We're storing in the + format so the user will see e.g. +9************73. Could potentially cause confusion - especially if they entered as 0093...

Reply:
Fair point. Let's keep it as you have it.

Reply:
Added a case for when the mobile number is 4 or less